src/main/java/com/softserve/teamproject/repository/custom/UserRepositoryImpl.java

```diff
@@ -13,7 +14,7 @@ public UserRepositoryImpl() {
   }
 
   public User getUserByNickName(String nickName) {
-    return from(QUser.user).where(getByNickName(nickName)).fetchFirst();
+    User user = from(QUser.user).where(getByNickName(nickName)).fetchFirst();
+    return UserDto.setImageAsBase64(user);
```

Comment:
We are using getUserByNickName() in different cases where image is useless You must remove your logic from here.